docs/src/assets/citations.css

```diff
@@ -0,0 +1,17 @@
+.citation dl {
```

Comment:
Overall this looks good to me, but where do these CSS tweaks come from and why do we need them?

Reply:
It's from DocumenterCitations.jl's docs: https://juliadocs.org/DocumenterCitations.jl/dev/styling/ . I think it makes bibliography look nicer :slightly_smiling_face: .

Reply:
Ah ok, maybe we could add that as a comment in the CSS?

Reply:
Sure, I will add a comment.

Reply:
Done.